packages/bsky/src/subscription/repo.ts

```diff
@@ -222,17 +259,16 @@ async function getOps(msg: Message): Promise<PreparedWrite[]> {
       op.action === WriteOpAction.Update
     ) {
       assert(op.cid)
-      const cid = CID.parse(op.cid)
-      const record = car.blocks.get(cid)
+      const record = car.blocks.get(op.cid)
       assert(record)
       return {
         action:
           op.action === WriteOpAction.Create
             ? WriteOpAction.Create
             : WriteOpAction.Update,
-        cid,
-        record: cborDecode(record),
-        blobs: [], // @TODO(bsky) need to determine how the app-view provides URLs for processed blobs
+        cid: op.cid,
+        record: cborToLexRecord(record),
+        blobs: [],
```

Comment:
do we want the appview to use this info to warm the cache?

Reply:
I think so, yeah!  I will peel off an issue for that, if that's cool.  In a way it's simple, but I am not sure off the top of my head the perfect approach to make it as low-impact to the app view as possible.

Reply:
yup yup no need to get it in here 👍

Reply:
👉  https://github.com/bluesky-social/atproto/issues/754